cql3/expr/expression.cc

```diff
@@ -2349,5 +2349,67 @@ const column_value& get_the_only_column(const expression& e) {
 
     return *result;
 }
+
+
+bool schema_pos_column_definition_comparator::operator()(const column_definition *def1, const column_definition *def2) const {
+    auto column_pos = [](const column_definition* cdef) -> uint32_t {
+        if (cdef->is_primary_key()) {
+            return cdef->id;
+        }
+
+        return 123456789;
```

Comment:
let's either define a static constant, or use something like uint32_t::max, instead of a random numeric constant in the code.

After some git archaeology with `git blame`, I see that this code was created to mimic Cassandra, and Cassandra 3.11 actually uses a different constant, NO_POSITION: https://github.com/apache/cassandra/blob/bc8a2604718284b95da52205a9b95b8a69483661/src/java/org/apache/cassandra/config/ColumnDefinition.java#L79-L86 . Perhaps we should define one too, as uint32_t::max()

Reply:
Changed to `uint32_t::max()`.